saiph/projection_test.py

```diff
@@ -350,6 +350,55 @@ def test_get_variable_contributions_calls_correct_subfunction(
     projection.get_variable_contributions(model, quanti_df)
 
 
+def test_get_variable_contributions_sum_is_100_famd(mixed_df: pd.DataFrame) -> None:
+    model = fit(mixed_df)
+    contributions = projection.get_variable_contributions(model, mixed_df)
+    summed_contributions = contributions.sum(axis=0)
+    assert_series_equal(summed_contributions, pd.Series([100.0] * 3), check_index=False)
+
+
+def test_get_variable_contributions_sum_is_100_with_col_weights_1_famd(mixed_df: pd.DataFrame) -> None:
+    model = fit(mixed_df, col_weights=[1.0, 1.0])
+    contributions = projection.get_variable_contributions(model, mixed_df)
+    summed_contributions = contributions.sum(axis=0)
+    assert_series_equal(summed_contributions, pd.Series([100.0] * 3), check_index=False)
```

Comment:
It would be better if you would test `famd.get_variable_contributions` in `famd_test.py`, rather than `projection.get_variable_contributions`.
Some test already checks that `projection.get_variables_contributions` calls `famd.get_variable_contributions` (`test_get_variable_contributions_calls_correct_subfunction`)